src/main/cli/settings.c

```diff
@@ -504,6 +508,7 @@ static const char * const lookupTableOsdLogoOnArming[] = {
 
 const lookupTableEntry_t lookupTables[] = {
     LOOKUP_TABLE_ENTRY(lookupTableOffOn),
+    LOOKUP_TABLE_ENTRY(lookupTableOsdUnit),
     LOOKUP_TABLE_ENTRY(lookupTableUnit),
```

Comment:
My only doubt is if we must maintain both, the Osd Unit and the Unit. 

I have not made a deep search, but it seems to me that the second one is only used in `frsky_unit`, so maybe we can modify it to accept the new BRITISH value too? Is an strange code, it gets the satellite quality signal, and transforms the units' I'm not too sure to what, and sends it in a TEMPERATURE field.

https://github.com/betaflight/betaflight/blob/b1af9befe547204fc46e902e064371ce04cd1399/src/main/telemetry/frsky_hub.c#L284-L301

Before changing nothing, let's see what others think.

Reply:
I explicit made a new osd unit for not having to touch the frsky telemetry. Not having to do so would be easier and more maintainable. Otherwise it would change in opentx, companion and maybe lua.

Reply:
I doubt it is used at anything more that here. Is not at MSP it any other place.
If not, I will not call it OSD, maybe unit (to be used in general in Betaflight since now) and the other unit_frsky if will only be used there.
But let's wait to @mikeller opinion.

Reply:
@McGiverGim You are right. There are no further references to the `frsky_unit`.
To make this right it could be refactored by merging the two existing parameters `frsky_unit` and `osd_units` into a more global name like `units_of_measure` ? 

Suggestion about naming are welcome.

Reply:
Yes, I think we can simplify this here, and use the metric / imperial / british enum for `frsky_unit`, and use 'metric' for 'british' in `frsky_hub.c`, since HDOP is a measure of distance.
Not sure if we shouldn't even go to the point of removing `frsky_unit` altogether and using `osd_units` (potentially with a rename) in its place - it seems to be unreasonable to have a dedicated configuration setting for something that has a very limited impact.

Reply:
Agree. At first I would not want to touch the telemetry part, but the impact was smaller than I thought. I will remove the added lookup table and change the HDOP message to adapt the new unit.
Variables `osd_units` and `frsky_unit` will use the same table.

Pfff, I was more afraid I had to make unit a global variable ;-)